src/pages/en/install/manual.mdx

```diff
@@ -4,21 +4,23 @@ description: 'How to install Astro manually with NPM, PNPM, or Yarn.'
 layout: ~/layouts/MainLayout.astro
 i18nReady: true
 ---
+import Button from '~/components/Button.astro'
 import FileTree from '~/components/FileTree.astro';
-import InstallGuideTabGroup from '~/components/TabGroup/InstallGuideTabGroup.astro';
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
 
-Ready to install Astro? Follow our automatic or manual set-up guide to get started.
+Ready to install Astro? Follow this manual set-up guide to get started.
```

Comment:
```suggestion
This guide will walk you through the steps to manually install and configure a new Astro project.
```
I'm thinking something less "encouraging" here, to nudge people towards the wizard unless they're darn sure they want to go through the manual process.

Reply:
Would this be overkill? I’m cool with your version if so.

```suggestion
This guide will walk you through the steps to manually install and configure a new Astro project if you prefer not to use [the automatic CLI tool](/en/install/auto/).
```